### DISCUSSION TITLE
Support continuous profiling

### DISCUSSION THREAD

**Discussion by dm3ch** [grafana/grafana#39301] [Observability]
For now continuous profiling is not yet widely adopted in community.
This methodology could be very useful sometimes to analyze application instance perfomance issues.

I believe that it would be very useful to support profiles visualisation in Grafana.
But I understand that continuous profiling is not so popular yet, so I suppose Grafana wouldn't adopt that for a long period of time.

Right now continuous profiling is supported by Google Stackdriver and DatadogHQ.
There's an open source continuous profiling system - https://github.com/conprof/conprof .

**Comment by dm3ch** [comment]
Grafana now supports phlare - their own continuous profiling storage. So 85's finally done.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
